server/polar/subscription/service.py

```diff
@@ -783,26 +800,90 @@ async def update_product(
         subscription.recurring_interval = product.recurring_interval
 
         if proration_behavior is None:
-            organization_repository = OrganizationRepository.from_session(session)
-            organization = await organization_repository.get_by_id(
-                product.organization_id
-            )
-            assert organization is not None
             proration_behavior = organization.proration_behavior
 
-        await stripe_service.update_subscription_price(
-            subscription.stripe_subscription_id,
-            new_prices=[
-                price.stripe_price_id for price in prices if is_static_price(price)
-            ],
-            proration_behavior=proration_behavior.to_stripe(),
-            metadata={
-                "type": ProductType.product,
-                "product_id": str(product.id),
-            },
-        )
+        if organization.subscriptions_billing_engine is False:
```

Comment:
That's related to your question

> How do we resolve contention between a Subscription having stripe_subscription_id not None vs. the Organization having subscriptions_billing_engine is True? If the subscription has stripe_subscription_id one would think it should go through Stripe, but the org is "saying" something else.

My approach on this would be: just rely on `subscription.stripe_subscription_id`, and don't care about the organization setting at this point. I'm quite convinced we'll have situations with organizations that have both subscriptions on Stripe and managed by us.

Reply:
Done 👍